### DISCUSSION TITLE
Skewed Arch block

### DISCUSSION THREAD

**Discussion by silvernull** [tsunamayo/Starship-EVO#4529] [Suggestions]
An arch block where it skews in another direction, since it is one of the few remaining shapes that would reduce the need for child entities to be used instead.
(shitty mockup)
![image](https://user-images.githubusercontent.com/80038133/152160453-af9e2b92-e062-4b03-8d4a-e0295e9752e4.png)

**Comment by tsunamayo** [comment]
@silverszi Hi, I didnt understood the shape sorry!
